rocAL/rocAL/source/pipeline/ring_buffer.cpp

```diff
@@ -147,8 +166,8 @@ void RingBuffer::init(RocalMemType mem_type, void *devres, std::vector<size_t> &
         {
             cl_mem_flags flags = CL_MEM_READ_ONLY;
 
-            _dev_sub_buffer[buffIdx].resize(sub_buffer_count);
-            for(unsigned sub_idx = 0; sub_idx < sub_buffer_count; sub_idx++)
+            _dev_sub_buffer[buffIdx].resize(_sub_buffer_count);
+            for(unsigned sub_idx = 0; sub_idx < _sub_buffer_count; sub_idx++)
```

Comment:
For OCL part we have not allocated ROI buffers

Reply:
Added